client/post-editor/editor-drawer/index.jsx

```diff
@@ -139,7 +139,10 @@ const EditorDrawer = React.createClass( {
 
 	renderSharing: function() {
 		return (
-			<EditorSharingAccordion
+			<CodeSplitRender
+				require={ function( callback ) {
+					require( [ 'post-editor/editor-sharing/accordion' ], callback );
```

Comment:
Why does this work? As you stated, webpack only does static analysis, but this would imply that is can do dynamic analysis... i think. Usually webpack is using require.ensure to inject split points.. plain requires, even hidden in functions, should be treated as hard deps that need to be in the same chunk (or a parent chunk)..